build.gradle

```diff
@@ -465,6 +465,23 @@ configure(rootProject) {
 		doFirst {
 			classpath = files(subprojects.collect { it.sourceSets.main.compileClasspath })
 		}
+		doLast {
+			copy {
+				from("$buildDir/docs") {
+					filter { String line ->
+						line.startsWith("<head>") ? line.replace("<head>", "<head><link rel=\"icon\" type=\"image/png\" href=http://junit.org/junit5/assets/img/junit5-logo.png>") :
+								line
+					}
+				}
+				into "$buildDir/temp"
+				include 'javadoc/**'
+			}
+			copy {
+				from "$buildDir/temp"
+				into "$buildDir/docs"
+			}
```

Comment:
It also seemed strange to me, especially when I would get the correct output in this form and empty files when I would use only one copy operation with the same input and output. The output of the documentation is generated by running all jar tasks which write to the documentation folder.